docs/source/ruleset.rst

```diff
@@ -0,0 +1,70 @@
+=======
+Ruleset
+=======
+
+This chapter explains how to use and take care of the ruleset. We assume you
+have peekaboo up and running and want to tweak or understand the default
+ruleset.
+
+We also asume you are familiar with python config parser.
+
+Section: rules
+==============
+Here rules can be disables by putting a ``#`` (comment) in front. Also the 
+order in which the rules will be processed can be changed by changing how
+the rules are listed (note that the trailing number is not relevant).
+
+
+Following sections
+==================
+The following sections are processed (if enabled in rules section) and
+contain for example the whitelist mime types. Comments also here can disables
+entries.
+
+Expressions
+===========
+Expressions are evaluated.
+It is a lot like Python itself.
+
+They can contain operators:
+``+ - * ** / // % << >> . < <= > >= == != in not in is is not isdisjoint and or``
+
+Datatypes are:
+``boolean, integer, real, string, regex, identifier, result``
+
+Rules can then be constructed like:
+.. code-block:: shell
+
+    expression.1  : sample.mimetypes <= {'text/plain', 'inode/x-empty', 'image/jpeg'} -> ignore
+    expression.2  : sample.meta_info_name_declared == 'smime.p7s'
+                        and sample.meta_info_type_declared in {
+                            'application/pkcs7-signature',
+                            'application/x-pkcs7-signature',
+                            'application/pkcs7-mime',
+                            'application/x-pkcs7-mime'
+                        } -> ignore
+    expression.3  : /DDE/ in cuckooreport.signature_descriptions -> bad
+
```

Comment:
Perhaps a terminology tweak:
* rule -> a rule of the ruleset, e.g. evil_sig or expression
* expression -> a expression of the expression rule
* condition -> the condition before `->`

?

Expressions will be tried one after another until one matches. The general structure of an expression is: `<condition> -> <result>`. If condition evaluates to true, the expression will be considered matching and result will be returned by the rule.

Possible results are: unknown, ignore, good and bad. The latter three will terminate ruleset processing and use the result as final analysis result while the former will continue on with the next rule of the ruleset (note2self: Will it?).

Reply:
I copied that. Thx

Reply:
Could perhaps move from the Terminology into the Expressions section and replace the opener "Expressions are evaluated" there?